### PR TITLE
Fetch Approved Liberal Studies Courses

### PR DESCRIPTION
**Pull request by mirandayu131**
Created a script that will (1) fetch the Other Yes List from its source, (2) parse it into a `CourseWithId[]`, and (3) write these courses to a JSON file to inspect later. 

Test Plan
- Test URL is https://apps.engineering.cornell.edu/liberalstudies/ApprovedLiberalStudies.pdf

### CODE REVIEW

**Review by elizabeth-tang — COMMENTED** [review]
Great job Miranda!! I'm curious about your use of try catch with an empty catch in parse.ts. Just a thought, what would it mean if an exception does occur during genCourseEntries and would you possibly want to modify/rerun without the offending part of the data?

File: package.json [hunk 1]
```diff
@@ -69,6 +69,8 @@
     "eslint-plugin-vue": "^8.0.3",
     "jest": "^26.6.3",
     "minimist": "^1.2.6",
+    "pdf-to-excel": "^0.3.2",
+    "pdfjs-dist": "^3.4.120",
```
Comment (zachary0kent) [inline package.json:73]: Do we use this dependency anywhere? If not, can we delete it?

**Review by zachary0kent — CHANGES_REQUESTED** [review]
Looks good, except one nit!

**Review by zachary0kent — CHANGES_REQUESTED** [review]
Also, can we modify the script to put the json in `src/assets/courses`? That's where our other json's with course information are located.

File: package.json (hunk 1 above)
Comment (mirandayu131) [reply package.json:73]: We use pdf-to-excel but not pdfjs-dist so I removed that

**Review by zachary0kent — CHANGES_REQUESTED** [review]
Can we also move the existing json to `src/assets/courses`? Thanks!

**Review by zachary0kent — APPROVED** [review]
Looks good!

**Pull request merged**